tests/opentelemetry_setup/otlp.yml

```diff
@@ -0,0 +1,25 @@
+mode: deployment
```

Comment:
Even though having a file is good enough, could you place these required files as constants inside the test case? It's more consistent with current approach and also helps identifying where are used